pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -41,7 +51,7 @@ func NewConnectivityCheckController(
 	c.Controller = factory.New().
 		WithSync(c.Sync).
 		WithInformers(allTriggers...).
-		ToController("ConnectivityCheckController", recorder.WithComponentSuffix("connectivity-check-controller"))
```

Comment:
is there are default sync interval? Otherwise the new CRD won't trigger sync here.

Reply:
Added crd trigger.